www/pushlink.js

```diff
@@ -44,19 +44,32 @@ var PushLink = function() {
 
 /**
  * Starts PushLink connection
+ * @param {string} packageName - The package name for the app (optional)
  * @param {string} apiKey - Your PushLink API key
  * @param {string} deviceId - The device id
  * @param {function} successCallback - A function to be called if the command succeeded
  * @param {function} errorCallback - A function to be called if the command failed
  */
-PushLink.prototype.start = function(apiKey, deviceId, successCallback, errorCallback) {
+PushLink.prototype.start = function(packageName,apiKey, deviceId, successCallback, errorCallback) {
   'use strict';
 
+  if (!errorCallback){
```

Comment:
What if `errorCallback` is `null`? It may be better to check for the length of `arguments` here.